src/main/resources/application-prod.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
Usually, production profile properties are passed via ENVIRONMENT VARIABLES to pass sensitive config props or to override any other default properties defined in `application.properties` file. So, I think this file is not required yet.